### PR TITLE
Removed existential deposit

### PR DESCRIPTION
**Pull request by clostao**
## Description

Frontier has a field called `ExistentialDeposit` in `pallet_balances::Config` that would behave as a unique OnCreation fee. I've updated this field to zero.

## Motivation

Balance of new accounts would be reducted their balance by `ExistentialDeposit`, this is not a desired behaviour.

## Impact on the project

`runtime/lib.rs`

### CODE REVIEW

**Review by GabrielMartinezRodriguez — APPROVED** [review]

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -251,7 +251,7 @@ impl pallet_timestamp::Config for Runtime {
 }
 
 parameter_types! {
-    pub const ExistentialDeposit: u128 = 500;
+    pub const ExistentialDeposit: u128 = 0;
```
Comment (deblanco) [inline runtime/src/lib.rs:254]: Because this is a spec of our chain, could you move the value to our cfg file?

**Review by deblanco — CHANGES_REQUESTED** [review]

File: runtime/src/lib.rs (hunk 1 above)
Comment (clostao) [reply runtime/src/lib.rs]: done

**Review by deblanco — CHANGES_REQUESTED** [review]
https://github.com/stabilityprotocol/stability/blob/main/docs/PARAMETERS.md

Could you add info about this change in the PARAMETERS doc?

**Review by deblanco — APPROVED** [review]
LGTM!

**Pull request merged**